README.pt.md

```diff
@@ -1,94 +1,99 @@
+## Assinaturas dos lançamentos
 
-*Releases* são assinadas com a chave
-[B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A)
-e publicadas [no GitHub](https://github.com/swaywm/sway/releases).
+Cada versão é assinada com a chave [B22DA89A][sway pgp] e publicada
+[no Github][sway releases].
 
 ## Instalação
 
 ### A partir de pacotes
 
-Sway está disponível em várias distribuições. Verifique se o pacote "sway" está
-disponível a partir do gerenciador de pacotes da sua distribuição. Caso não esteja,
-procure por informações sobre como instalar o Sway na sua distribuição
-[aqui](https://github.com/swaywm/sway/wiki/Unsupported-packages).
+Sway está disponível em várias distribuições. Verifique se o pacote
+"sway" está disponível para ser instalado na sua distribuição GNU/Linux. Caso não esteja,
+procure por informações sobre como instalar Sway na sua distribuição
+[aqui][sway pkg wiki].
 
-Se você está interessado em manter um pacote do Sway para a sua distribuição,
-visite o canal no IRC ou mande um email para sir@cmpwn.com (*em inglês*).
+Se estiveres interessado em manter um pacote em sua distribuição, visite o
+canal no IRC ou mande um e-mail para <sir@cmpwn.com> (*em inglês*).
 
 ### A partir do código-fonte
 
-Antes de iniciar a compilação, instale as dependências:
+Instale as seguintes dependências:
 
-* meson
-* [wlc](https://github.com/Cloudef/wlc)
+* meson *
+* [wlroots][]
 * wayland
-* xwayland
-* libinput >= 1.6.0
-* libcap
+* wayland-protocols *
 * pcre
-* json-c >= 0.13
+* json-c
 * pango
 * cairo
-* gdk-pixbuf2 *
-* pam **
-* [scdoc](https://git.sr.ht/~sircmpwn/scdoc) (man pages)
-* git
+* gdk-pixbuf2 **
+* pam (opcional: Para suporte PAM  para o swaylock)
+* [scdoc][] (opcional: Para as páginas de manuais)
+* git *
 
-_\*Dependência apenas de swaybar, swaybg, e swaylock_
+_\* Dependências em tempo de compilação
 
-_\*\*Dependência apenas de swaylock_
+_\*\*opcional: Necessário apenas para swaybg, e swaylock_
 
-Para compilar, execute estes comandos:
+Execute os seguintes comandos:
 
     meson build
     ninja -C build
     sudo ninja -C build install
 
-Em sistemas com logind, configure as seguintes capacidades para o arquivo binário:
-
-    sudo setcap "cap_sys_ptrace,cap_sys_tty_config=eip" /usr/local/bin/sway
-
-Em sistemas sem logind, ative a *flag* de *suid* do arquivo binário:
+Em sistemas sem logind, você precisa ativar as *flags* de suid no executável gerado:
 
     sudo chmod a+s /usr/local/bin/sway
 
+Sway removerá as permissões de root logo após o início do processo.
+
 ## Configuração
 
-Se você já usa o i3, copie o arquivo de configuração do i3 para `~/.config/sway/config`;
-o Sway lerá o arquivo normalmente. Senão, copie o arquivo de configuração de exemplo
-para `~/.config/sway/config`. É comum esse arquivo estar localizado em
-`/etc/sway/config`. Veja `man 5 sway` para informações sobre configuração.
+Se já usas i3, copie teu arquivo de configuração para `~/.config/sway/config`
```

Comment:
`Se você já usa i3, copie sua configuração para ...`

Reply:
I think `configuração` already strongly implies that it's an `arquivo` so it becomes redundant.